src/Sarif/Writers/ResultLogJsonWriter.cs

```diff
@@ -364,7 +364,7 @@ private void EnsureInitialized()
         {
             if (_writeConditions == Conditions.None)
             {
-                Initialize(id : Guid.NewGuid().ToString(), correlationId: null);
+                Initialize(id : Guid.NewGuid().ToString(), automationId: null);
```

Comment:
>automationId [](start = 59, length = 12)

my final 'fine-tooth comb' review of spec vs. sdk missed this oversight. tsk. :)